assets/src/styles/blocks/DeepDiveTopic.scss

```diff
@@ -0,0 +1,25 @@
+.deep-dive-topic {
```

Comment:
I need to add this styles since the editor does not provide some of them, like `box-shadow`, `min-height`, among others.

c.c. @mleray

Reply:
We can leave this customizations out of this implementation for now, if there is no other way of applying them, and let the design team know. Most of these patterns are under assessment as pilots will start using them. So better not add any specific css classes just yet.